blscosi/bdnproto/struct.go

```diff
@@ -0,0 +1,67 @@
+package bdnproto
+
+import (
+	"errors"
+	"fmt"
+
+	"go.dedis.ch/cothority/v3/blscosi/protocol"
+	"go.dedis.ch/kyber/v3"
+	"go.dedis.ch/kyber/v3/pairing"
+	"go.dedis.ch/kyber/v3/sign"
+	"go.dedis.ch/kyber/v3/sign/bdn"
+	"go.dedis.ch/kyber/v3/sign/cosi"
+)
+
+// BdnSignature is a signature that must be verified using coefficients
+// derived from the public keys
+type BdnSignature []byte
+
+// GetMask returns the bytes representing the mask
```

Comment:
GetMask does not return bytes. Maybe just copy and paste the other GetMask doc string, which I also corrected slightly (see below).